help/text-emphasis.md

```diff
@@ -0,0 +1,45 @@
+# Text emphasis
+
+Zulip supports Markdown syntax for bold, italic, and strikethrough text.
+
+{!emphasis.md!}
+
+## Using the bold text button
+
```

Comment:
This heading feels odd to me... Mostly instruction blocks have a heading that describes what the user wants to do, which in this case is making text bold, not using a particular button.

What if we restructured this page to with separate sections for bold, italics and strikethrough, and tabs for "Via Markdown", "Via compose box buttons", or something along those lines?

If it works well, I think having the button instructions appear on the Message formatting page too would be nice.

Reply:
Yeah, let's try that.

Reply:
I think it works well. I also like that we can use the include block from "Message formatting" as a quick summary at the top of each page. What do you think?

<img width="500" alt="image" src="https://user-images.githubusercontent.com/2343554/218235001-f969ae07-3d55-4f4f-8c89-26b9ad479dc1.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/2343554/218234831-ad8afb2e-21e4-4cce-945b-5de74c4b5c93.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/2343554/218234897-19b3c8ec-acbb-4406-895b-c9c8b0b86401.png">